tests/test_dashboard/global.R

```diff
@@ -2,3 +2,5 @@ library(shiny)
 devtools::load_all(path = "../../")
 
 ga_key <- "ABCDE12345"
+
+bookmarking_whitelist <- c("navlistPanel", "tabsetpanels")
```

Comment:
Missing a line here for it to work:
```suggestion
bookmarking_whitelist <- c("navlistPanel", "tabsetpanels")
enableBookmarking("url")
```